plugins/foreman_salt/3.0/index.md

```diff
@@ -76,7 +76,7 @@ If using the Foreman installer, install the core and smart proxy plugins with:
 foreman-installer --enable-foreman-plugin-salt --enable-foreman-proxy-plugin-salt
 ```
 
-For manual installation, see [How to Install a Plugin](http://www.theforeman.org/manuals/latest/index.html#6.1InstallaPlugin) for more info.
+For manual installation, see the [plugin manual](http://www.theforeman.org/plugins/#2.Installation) for more info.
```

Comment:
Could you remove the domain as well please?  `{{site.baseurl}}plugins/..` ought to work.